go/dism/dism.go

```diff
@@ -53,7 +56,10 @@ const (
 
 // Session holds a dism session. You must call Close() to free up the session upon completion.
 type Session struct {
-	Handle uint32
+	Handle         *uint32
```

Comment:
Was there a particular reason for changing this to a pointer?

Reply:
I needed to change the Handle to a pointer so that the call defer `dismSession.Close()` on the caller side doesn't close the initial handle again (already closed if a session reload happend) but instead closes the new handle (the new one created by the reload).